code/game/jobs/whitelist.dm

```diff
@@ -10,7 +10,8 @@ GLOBAL_LIST_EMPTY(whitelist)
 
 /proc/load_whitelist()
 	GLOB.whitelist = file2list(WHITELISTFILE)
-	if(!GLOB.whitelist.len)	GLOB.whitelist = null
+	if(!GLOB.whitelist.len)
```

Comment:
```suggestion
	if(!length(GLOB.whitelist))
```

Reply:
Outside scope. This PR is only concerned with the actual query logic, not the entirety of the surrounding proc.